core/controllers/reader.py

```diff
@@ -291,7 +348,12 @@ class StateHitEventHandler(base.BaseHandler):
 
     @require_playable
     def post(self, exploration_id):
-        """Handles POST requests."""
+        """Handles POST requests.
+        Records learner's new state
```

Comment:
Put this on the end of the previous line, and change to: "Records that a learner has encountered a new state in the exploration."